### PR TITLE
Changed error messages layout

### PR DESCRIPTION
**Pull request by joeycatxx**
**Initially display**
errors

**when you click show all**
errors-show-all

### CODE REVIEW

File: src/app/exercise/runner/runner.component.ts [hunk 1]
```diff
@@ -235,15 +235,50 @@ function injectIframe(element: any, config: IframeConfig, runner: RunnerComponen
                 font-weight: 300;
                 background-color: #ffe;
               }
+              .mainerror {
+                color: black; 
+                font-size: larger;
+              }
+              .othererrors {
```
Comment (prestonvanloon) [inline src/app/exercise/runner/runner.component.ts]: What is the difference between main errors and other errors? They look the same?

File: src/app/exercise/runner/runner.component.ts [hunk 2]
```diff
@@ -235,15 +235,50 @@ function injectIframe(element: any, config: IframeConfig, runner: RunnerComponen
...
-<h2>Errors when compiling</h2>
- <div>Look in the editor for hints to fix it.</div>
- <ul>
-  ${diagnosticsHtml}
- </ul>`);
+            console.log(diagnostics);
+
+            const errList = diagnostics.map( (diagnostic, i) => {
+              if (i === 0) {
+                return `<li style="margin-bottom: 10px;"><div class="mainerror">Error in <b>${diagnostic.file.fileName}</b><br />` +
```
Comment (prestonvanloon) [inline src/app/exercise/runner/runner.component.ts]: You wouldn't need this if block to add the margin-bottom if you use CSS for the `ul`.

```css
ul.error-list li:first-child {
   margin-bottom: 10px;
}
```

```html 
<ul class="error-list">
   <li>This has margin bottom 10px</li>
   <li>This does not</li>
</ul>
```

File: src/app/exercise/runner/runner.component.ts (hunk 1 above)
Comment (joeycatxx) [inline src/app/exercise/runner/runner.component.ts]: Seems the changes were required in a different area. Kirill told me he would cancel this change request.

**Comment by kirjs** [comment]
closing,

**Pull request closed without merging**